editor/import/resource_importer_csv_translation.cpp

```diff
@@ -147,6 +147,9 @@ Error ResourceImporterCSVTranslation::import(const String &p_source_file, const
 		if (r_gen_files) {
 			r_gen_files->push_back(save_path);
 		}
+
+		ResourceUID::ID save_id = hash64_murmur3_64(translations[i]->get_locale().hash64(), p_source_id);
```

Comment:
Shouldn't we put the path to the hash? I'm wondering if there's two identical translation files, what happens?

Reply:
It hashes both the UID of the file and the language, so this should be unique.

Reply:
One of the biggest features of uid is that you can relocate the files. so hashing the path would remove that.

Reply:
It does not matter, because it uses the base UID for this, which is generated once and then kept if you move it.
It should continue working fine even if you move the file.